examples/resources/blameless_incident_type/resource.tf

```diff
@@ -25,15 +25,26 @@ resource "blameless_incident_type" "incident_type_security" {
     ]
 
     # MS Teams Notifications
-    teams_invited_users = [
-      "@dillon"
-    ]
-    teams_announcement_groups = [
-      "@responders"
-    ]
-    teams_announcement_channels = [
-      "#general"
-    ]
+    teams_announcement_group = "fa1a4481-4ba9-45df-96b9-71c48163bd56"
+
+    teams_invited_users = {
+      id           = "8d8c2f9b-c341-44e1-b5cd-31c210fb64f8"
+      display_name = "Pablo"
+    }
+
+    teams_announcement_channels = {
+      id           = "19:knHgmGNOCmFMKAKaJJtqb2KwaonhzOEf66Oj7y0Ljn41@thread.tacv2"
+      display_name = "General"
+      group_id     = "fa1a4481-4ba9-45df-96b9-71c48163bd56"
+      group_name   = "Blameless"
```

Comment:
Same question as above. This seems like it would be difficult for a normal user to get this information, much less maintain it.

Reply:
We've decided to resolve this in a following PR.